flink-table/flink-table-runtime/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,10 @@
+flink-table-runtime
+Copyright 2014-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the BSD license.
+See bundled license files for details
+
+- org.codehaus.janino:janino:3.0.7
```

Comment:
this is incorrect as flink-table-runtime isn't bundling any dependencies (as in, it is not included in the jar).
Which raises the question: which module is now bundling janino?